### ISSUE TITLE
Accessibility issue with kirby-checkbox - Missing accessible name even with aria-label set

### ISSUE THREAD

**Issue by fr1m3r** [kirbydesign/designsystem#3843]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
The Level Access tool is reporting our use of kirby-checkbox on several of our consents in DRB, regarding the missing accessible name.
However, when we add a label using aria-label or [attr.aria-label] regarding the missing accessible name, the Level Access tool says that we still have an issue with this.

## Describe how to reproduce the bug
1. Login to DRB Web (no matter which bank)
2. Go to 'Profile' in the left menu
3. Go to 'Consent and agreements'
4. Go to the consent called 'Marketing consent'
5. Scan the 'Marketing consent' page with the Level Access tool (extension to Chrome)
6. See the reported issue in the Level Access tool

## Which Kirby version was used?
10.2.1

## What was the expected behavior?
That the text attribute sets the accessible name or that the addition of aria-label or [attr.aria-label] will add the accessible name.

## Add any screenshots

![Image](https://github.com/user-attachments/assets/dc90fdba-e71c-4c52-a1fa-4cb1d06fbb9b)
![Image](https://github.com/user-attachments/assets/d1a53050-1578-4c05-b93a-8a6ff464de0a)

## Please complete the following information:
- OS: [Windows 11]
- Browser [Chrome]
- Browser version [133.0.6943.127]

## Are there any additional context?
No

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
@fr1m3r thank you for your issue! This is only an issue with the nested SVG **_inside_** the checkbox, and not the checkbox itself. Your checkbox most likely has an accessible name based on the label, and there should be no need for adding an `aria-label`.

There is of course still an issue here, but it is most likely much less critical than anticipated - and I have forwarded the issue to ionic, where the root cause seems to lie.

https://github.com/ionic-team/ionic-framework/issues/30231

**Comment by RasmusKjeldgaard** [comment]
The underlying issue is solved by Ionic, and we should now update to [v8.7.3](https://github.com/ionic-team/ionic-framework/releases/tag/v8.7.3) 🎉 

Consider also removing the stencil peer dependency while the package.json is open, as it is not needed.

**Comment by Loujuu** [comment]
Tested with Level Access Tool to reproduce the bug. The bug could not be reproduced. Task is closed.
